src/core/components/base-header.vue

```diff
@@ -14,6 +17,37 @@ const router = useRouter();
 const auth = useAuth();
 const user = ref(auth.user);
 
+const confirm = useConfirm();
+const toast = useToast();
+
+const handleConfirmation = () => {
+  confirm.require({
+    header: "Account verification",
+    message:
+      "Do you want your account to be verified by WAW? These action might take weeks and it's irreversible.",
```

Comment:
Could you make this message less like a dangerous action and more like an informative message about being verified?

Reply:
Uhmmmmm, that's odd. The double dialog problem doesn't happen in dev... 

Also yes, the messages will be friendlier